libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/scanners/cellscanner/CellScanner.java

```diff
@@ -54,6 +56,8 @@ public void start(final ActiveOrPassiveStumbling stumblingMode) {
             return;
         }
 
+        mPassiveScanCount.set(0);
```

Comment:
The idea is that the variable is member of the scanner so that it can be reset from outside the TimerTask.